static/src/javascripts/projects/common/modules/identity/upsell/follow/FollowButtonWrap.js

```diff
@@ -34,9 +34,9 @@ class FollowButtonWrap extends Component<
         } else {
             this.props.onUnfollow();
         }
-        this.setState(() => ({
```

Comment:
i realized i went a big dogmatic here before, functional setState is pointless if you don't need to use the current state or props